supervisord.conf

```diff
@@ -0,0 +1,8 @@
+[supervisord]
+nodaemon=true
+
+[program:gulp]
+command=/bin/bash -c "/crowbar-ui/node_modules/gulp/bin/gulp.js"
```

Comment:
Can I suggest the use of:

```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
```

on the program(s) section so we can see the output of all programs with docker logs $CONTAINER_ID?

Also, using `autorestart=true` may be good in case we break something and fix it so we dont need to restart the container and its restarted automagically :)